src/app/rpmostree-builtin-testutils.cxx

```diff
@@ -51,11 +52,11 @@ rpmostree_builtin_testutils (int argc, char **argv,
                                         invocation, cancellable, error);
   else
     {
-        g_autoptr(GPtrArray) args = g_ptr_array_new ();
-        for (int i = 0; i < argc; i++)
-          g_ptr_array_add (args, argv[i]);
-        g_ptr_array_add (args, NULL);
-        return ror_testutils_entrypoint ((char**)args->pdata, error);
+      rust::Vec<rust::String> rustargv;
+      for (int i = 1; i < argc; i++)
```

Comment:
Shouldn't this still be 0?

Reply:
Yep thanks!  Looks like it's different from cliwrap.